engine/crates/composition/tests/composition/composed_directives_basic/federated.graphql

```diff
@@ -54,8 +54,8 @@ type ObserverDetails {
     observerType: ObserverType! @join__field(graph: OBSERVATIONS)
 }
 
-type BirdSighting {
-    bird: Bird! @join__field(graph: SIGHTINGS)
+type BirdSighting @deprecated(reason: "we haven't seen any birds in a while :(") {
```

Comment:
Looks like we were missing deprecated here. Thanks to the recent reorganization that is prolonged in this PR, handling of directives is uniform across locations and things like that are less likely to happen.